inference-engine/tests/functional/plugin/shared/include/behavior/set_blob_of_kind.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "common_test_utils/common_utils.hpp"
+
+namespace BehaviorTestsDefinitions {
+
+enum class BlobKind {
```

Comment:
I think, we can reuse this functionality in the future. Could you move `BlobKind`, `std::ostream& operator<<` and `makeBlobOfKind` to [the data utils](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/ie_test_utils/functional_test_utils/blob_utils.hpp) ?

Reply:
Unfortunately, this `enum` is not reusable as-is - it covers kinds related to this particular change only, but IE defines much bigger set of blob kinds.

Propose to design and implement reusable blobs kind test utilities out of  this change which introduces just `BatchedBlob`.

Reply:
I can extend it later. I offer to move the existing functionality

Reply:
done.